tests/unit/checkmate/views/status_test.py

```diff
@@ -0,0 +1,12 @@
+from h_matchers import Any
+
+from checkmate.views.status import get_status
+
+
+class TestStatusRoute:
+    def test_status_returns_a_response(self, make_request):
+        request = make_request("/status")
+
+        result = get_status(request)
+
+        assert result == Any.dict()
```

Comment:
This is intentionally vague. As long as the response is successful the payload actually isn't important. We plan to start sending back stuff like "degraded" and "last_update_time" and stuff if we can, at which point we probably care quite a bit what's in here.